src/v3/src/transformer/button/transformHelpLinks.ts

```diff
@@ -24,42 +24,43 @@ export const transformHelpLinks: TransformStepFnWithOptions = ({
 }) => (
   formbag,
 ) => {
-  const shouldAddButton = transaction.availableSteps
-    ?.some(({ name }) => STEPS_REQUIRING_HELP_LINK.includes(name));
-  if (!shouldAddButton) {
-    return formbag;
-  }
+    const shouldAddButton = transaction.availableSteps
+      ?.some(({ name }) => STEPS_REQUIRING_HELP_LINK.includes(name));
+    if (!shouldAddButton) {
+      return formbag;
+    }
 
-  const helpLinkHref = getHelpLink(widgetProps);
-  const helpLink: LinkElement = {
-    type: 'Link',
-    contentType: 'footer',
-    options: {
-      href: helpLinkHref,
-      step: '',
-      label: loc('help', 'login'),
-      dataSe: 'help',
-    },
-  };
+    const helpLinkHref = getHelpLink(widgetProps);
+    const helpLink: LinkElement = {
+      type: 'Link',
+      contentType: 'footer',
+      options: {
+        href: helpLinkHref,
+        step: '',
+        label: loc('help', 'login'),
+        dataSe: 'help',
+        target: '_blank',
+      },
+    };
 
-  const transformCustomLink: (link: CustomLink) => LinkElement = ({ href, text, target }) => ({
-    type: 'Link',
-    contentType: 'footer',
-    options: {
-      href,
-      target,
-      step: '',
-      label: text,
-      dataSe: 'custom',
-    },
-  });
-  const customHelpLinks = getCustomHelpLinks(widgetProps).map(transformCustomLink);
+    const transformCustomLink: (link: CustomLink) => LinkElement = ({ href, text, target }) => ({
+      type: 'Link',
+      contentType: 'footer',
+      options: {
+        href,
+        target,
+        step: '',
+        label: text,
+        dataSe: 'custom',
+      },
+    });
+    const customHelpLinks = getCustomHelpLinks(widgetProps).map(transformCustomLink);
```

Comment:
Is the linter fine w/ all of these changes in indentation? I assume these are coming from your local indentation preferences or something?

Reply:
no lint doesn't like it. not sure what config is kicking in to apply that change. fixing now